app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -1015,7 +1015,7 @@ private void openZimFile(File file, boolean clearHistory) {
       ActivityCompat.requestPermissions(this,
           new String[] { Manifest.permission.READ_EXTERNAL_STORAGE },
           REQUEST_STORAGE_PERMISSION);
-      if (BuildConfig.IS_CUSTOM_APP && Build.VERSION.SDK_INT == Build.VERSION_CODES.M) {
```

Comment:
You should revert this change and delete this line `import static android.os.Build.VERSION_CODES;` so that there is consistency in using version codes.

Reply:
Well, this line was there from before I cloned this repo. You can check that.